evergreen/eg_app/views.py

```diff
@@ -12,6 +18,54 @@
 def index(request):
     return render(request,'index.html')
 
+def getFileType(filePath):
+    return mimetypes.guess_type(filePath)[0]  
+
+def fileHandler(request, fileName): # can handle img and text 
+    # sanitizedfileName = quote(fileName) # get a clean full file path 
+    # path = Path(settings.STATIC_ROOT) / sanitizedfileName
+
+    path = fileName
+
+    allowedType = {'.css', '.html', '.js', '.png', '.jpg', '.jpeg', '.gif', '.mp3', '.mp4', '.xml', '.json', '.pdf','.ico'} # can add more, 
+
+    if not path.suffix.lower() in allowedType: # to deal with user uploads 
+        return HttpResponseNotFound("404 - File type not allowed")
+    
+    # #deal with /../ attacks 
+    # rootPath = Path(settings.STATIC_ROOT).resolve() #.resolve get absolute path 
+    # if not path.resolve().is_relative_to(rootPath):
+    #     return HttpResponseNotFound("404 - Invalid file path")
+    
+    if path.exists() and path.is_file(): # make sure its not a directory 
+        contentType = getFileType(path)
+        
+        try:
+            with open(path, 'rb') as file:
+                content = file.read()
+
+            response = HttpResponse(content, content_type=contentType)
+            response['X-Content-Type-Options'] = "nosniff"
+            #response['Content-Length'] = str(len(content)) is need for very large file, django handles is not
+
+            print(response)
```

Comment:
Why are we printing this to the terminal? Instead of doing printing for debugging (assuming this is what you are doing) it is probably better to use the logging module so we can turn these on and off.